app/gui2/src/assets/base.css

```diff
@@ -60,14 +59,42 @@
 
 body {
   min-height: 100vh;
+  /* TEMPORARY. Will be replaced with actual background when it is integrated with the dashboard. */
+  background: #e4d4be;
   color: var(--color-text);
-  background: var(--color-background);
-  transition: color 0.5s, background-color 0.5s;
-  line-height: 1.6;
-  font-family: Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu,
-    Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
-  font-size: 15px;
+  transition:
+    color 0.5s,
+    background-color 0.5s;
+  font-family:
+    Inter,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  font-size: 11.5px;
+  font-weight: 500;
+  line-height: 174.5%;
```

Comment:
Where does this come from?

Reply:
these values are taken directly from the figma design.

Reply:
Figma design doesn't define consistent line height for all text elements. There is a mix of `171.5%` and `174.5%`. I think it's just accidental and carries over through copy and paste. But I guess we can use the larger value here, as it is also used in the chat bubbles, the only multiline text in the design where it actually has any implications.